ShellScript/Shell-Unix-Generic.sublime-syntax

```diff
@@ -13,6 +13,22 @@ file_extensions:
   - .bash_profile
   - .bash_variables
   - .bashrc
+  - .eslintignore
+  - .git
+  - .git-blame
+  - .git-commit
+  - .git-graph
+  - .gitattributes
+  - .gitblame
+  - .gitcommit
+  - .gitconfig
+  - .gitgraph
+  - .gitignore
```

Comment:
Most of these are *not* shell syntax and instead something custom (more akin to TOML or INI). I do not think they should be here.